build.gradle.kts

```diff
@@ -9,10 +10,14 @@ repositories {
     mavenCentral()
 }
 
-dependencies {
-    testCompile("junit", "junit", "4.12")
+subprojects{
+    apply(plugin="java")
+    dependencies{
```

Comment:
Don't set dependencies from root project. Every project should explicitly list its dependencies.

Gradle documentation suggest to use 2 dependencies and add special clause to test configuration. Do exactly as is written there.